src/main.cpp

```diff
@@ -3757,7 +3759,12 @@ void static UpdateTip(CBlockIndex *pindexNew, const CChainParams& chainParams) {
     RenderPoolMetrics("sapling", saplingPool);
     RenderPoolMetrics("transparent", transparentPool);
 
-    cvBlockChange.notify_all();
+    {
+        boost::unique_lock<boost::mutex> lock(g_best_block_mutex);
```

Comment:
What is the difference between a `boost::unique_lock<boost::mutex>` and the `WaitableLock` type of the cherry-picked commit?